xml/System.IO/BinaryReader.xml

```diff
@@ -709,10 +709,12 @@
         <Parameter Name="buffer" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">A region of memory. When this method returns, the contents of this region are replaced by the bytes read from the current source.</param>
+        <summary>Reads a sequence of bytes from the current stream and advances the position within the stream by the number of bytes read.</summary>
```

Comment:
Essentially the same as calling Stream.Read(Span<byte>):
https://github.com/dotnet/corefx/blob/65edbb39dd737e8529a343f385d2600e482b7ff3/src/Common/src/CoreLib/System/IO/BinaryReader.cs#L477